### PR TITLE
apply good practices from NeoDocs

### PR DESCRIPTION
**Pull request by sebaswalden**
-Update packages
-Migrate from CommonJS to ESModules
-Separate Express app and Server
-Install ESLint security plugin
-Change validateFields middleware for a validateInput function that uses [AJV](https://ajv.js.org/)
-Change deprecated Sequelize import and moved `./src/models/index.js` to `./src/service/database.js`
-Add Winston for better logging
-Add .nvmrc
-Update README.md

Note: Only using Babel for Sequelize configuration.

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "presets": ["es2015"],
+  "plugins": [
+      "add-module-exports"
+  ]
+}
```
Comment (gonzasismo) [inline .babelrc:6]: Oops

File: .nvmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+v16.13.0
```
Comment (gonzasismo) [inline .nvmrc:1]:
```suggestion
v16.13.0

```

**Review by gonzasismo — COMMENTED** [review]
Great job! Left some comments

File: .babelrc (hunk 1 above)
Comment (sebaswalden) [reply .babelrc:6]: 😅

**Review by lilhand — APPROVED** [review]

**Pull request merged**